mobile/lib/profile/screens/profile_page.dart

```diff
@@ -135,72 +91,111 @@ class _ProfilePageState extends State<ProfilePage> {
+                                    textAlign: TextAlign.center,
+                                    style: const TextStyle(
+                                        fontSize: 20,
+                                        color: Colors.black,
                                         fontWeight: FontWeight.w300),
                                   ),
+                                )
+                              ],
+                            )),
+                      ),
+                      Padding(
+                        padding:
+                            EdgeInsets.symmetric(vertical: screenHeight * 0.01),
+                        child: Container(
+                          height: screenHeight * 0.5,
+                          decoration: BoxDecoration(
+                              color: Colors.grey[200],
+                              border: Border.all(
+                                color: Colors.black,
+                              ),
+                              borderRadius:
+                                  const BorderRadius.all(Radius.circular(30))),
+                          child: Column(
+                            children: [
+                              Padding(
+                                padding: EdgeInsets.symmetric(
+                                    vertical: screenHeight * 0.02),
+                                child: const Text(
+                                  'Statystyki',
+                                  textAlign: TextAlign.center,
+                                  style: TextStyle(
+                                      fontSize: 34,
+                                      color: Colors.black,
+                                      fontWeight: FontWeight.w300),
                                 ),
                               ),
-                            )),
-                      ],
-                    ),
+                              const CustomCounterDescription(
+                                  text: 'Liczba wykonanych ćwiczeń'),
+                              CustomCounter(
+                                  text: profilePageData.finishedActivitiesCount!
+                                      .toString()),
+                              const CustomCounterDescription(
+                                  text: 'Aktualny daily streak'),
+                              CustomCounter(
+                                  text: profilePageData.dailyStreak.toString()),
+                              const CustomCounterDescription(
+                                  text: 'Liczba znajomych'),
+                              CustomCounter(
+                                  text: profilePageData.friendsCount.toString())
+                            ],
+                          ),
+                        ),
+                      ),
+                      Padding(
+                          padding: EdgeInsets.symmetric(
```

Comment:
oddzielny widget byłby spoko, jak tej customizacji jest dużo to lepiej wywalić to do oddzielnego pliku